src/components/cardProjectDetail/CardProjectDetail.tsx

```diff
@@ -89,6 +90,12 @@ const CardProjectDetail: FC<Props> = ({
   organizedBy,
   rating,
   status,
+  follower,
```

Comment:
Instead of adding this 6 props that go directly to the Follow components, it would be better to reaceive a single object called followProps (with the type exported from Follow.types), and then spread it in the component:
<Follow {...followProps} />